### ISSUE TITLE
Failing to Import Test Module 

### ISSUE THREAD

**Issue by astone5793** [timduly4/pyglow#102]
Hi, 

I think I correctly installed Pyglow, but I'm getting an error when I try running the `python3 -m unittest test.test_suite_pyglow` command. The message I get is: 

```
ERROR: test_suite_pyglow (unittest.loader._FailedTest)

ImportError: Failed to import test module: test_suite_pyglow
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
ModuleNotFoundError: No module named 'test.test_suite_pyglow'

Ran 1 test in 0.000s

FAILED (errors=1)
051764:test labuser$ python3 -m unittest test.test_suite_pyglow

```

I made sure I ran `make -C src/pyglow/models source` first, and I checked `/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages` for the pyglow folder and .so files. They were not there, but they were in `/Users/labuser/Library/Python/3.7/lib/python/site-packages` I'm not sure if that matters. I tried moving them but that didn't help. 

I also tried to run the example scripts individually, but I get an error that says `Traceback (most recent call last):
  File "/Users/labuser/pyglow/examples/test_airglow.py", line 8, in <module>
    import pyglow
ImportError: No module named pyglow`

I'm using a Mac with the latest version of macOS Mojave.

Thanks for your help.

**Comment by timduly4** [comment]
@Orion5793 , this looks like pyglow was not installed successfully.  You can verify installation with:

```
$ python3 -c "import pyglow"
$
```

If pyglow was installed successfully, nothing will return.  What do you see on your end?

**Comment by Orion5793** [comment]
Hi, 

Thanks for the reply. 

I just ran the command in Terminal and nothing was returned. 

I am able to run the test_ne_airglow_profile.py file within my IDE, but when I run test_version_keywords.py, I get the error: `  File "/Users/labuser/pyglow/examples/test_version_keywords.py", line 14, in <module>
    pt._run_hwm93()
AttributeError: 'Point' object has no attribute '_run_hwm93'` So I think some of it is installed correctly, but not all of it.

**Comment by Orion5793** [comment]
Do you have any advice?

**Comment by bharding512** [comment]
Did you try both 
`python3 setup.py install --user`
and
`python3 setup.py install`

I would also try `python` instead of `python3`. 

I may be off-base, but it looks like your system may be getting mixed up between two installations of Python: one factory-installed (/Library/Frameworks/Library.Frameworks/...) and one installed some other way, in your user (/Users/labuser/Library/Python). If this is the case, I would recommend using virtual environments or anaconda to isolate your personal Python installation from the system's Python.

**Comment by Orion5793** [comment]
Sorry about that; I was out of the lab for a few weeks. 

I completely uninstalled it and started over. 

When using the `make all` I got a fair amount of errors. It's too long to copy and paste the whole thing here, so I've attached a screenshot of some of what I got. 

After that, I used `python3 setup.py`. I got lots of similar errors to the screenshot previously mentioned. However, I did manage to get the .io files to copy to `/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages` by not using the `--user` flair when setting it up.

This time, when I used `python3 -c "import pyglow"`, I got this output: 

`[generate_kpap.py] Generating geophysical indices table...
[generate_kpap.py] Generated: /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pyglow/geophysical_indices.npy` 

I ran the same command again and nothing was returned, so I think it worked. 

In my IDE, I was able to run all the .py files in the 'examples' folder, except 'test_version_keywords.py', which gave me the same error I mentioned in my last comment: 

`File "/Users/labuser/pyglow/examples/test_version_keywords.py", line 14, in <module> pt._run_hwm93() AttributeError: 'Point' object has no attribute '_run_hwm93'`

Thanks for your help. 

![Screen Shot 2019-05-13 at 4 17 16 PM](https://user-images.githubusercontent.com/11618499/57653013-db894600-759e-11e9-8553-32cedd16ae57.png)

**Comment by bharding512** [comment]
The output you showed aren't errors, just warnings. The underlying models print many warnings when compiled, but that doesn't mean something is wrong.

The test_version_keywords.py is a bug, thanks for noticing this. We should update this test to be consistent with the changes made in #72.

Other than that, is everything working ok?

**Comment by Orion5793** [comment]
Ah, I see, my bad. I'm pretty new to this sort of thing, thanks for your patience. 

And yep, everything else seems to be working!

**Comment by bharding512** [comment]
Reopening because test_version_keywords.py still has a bug wrt hwm call.

**Comment by sandyfreelance** [comment]
Might be a deprecated feature-- I also get the hwm bug in my new python3 install; all the other tests run fine via 'pytest'.  But if I comment out the run_hwmxx() calls and leave in the run_hwm() calls, the rest runs fine, i.e. the lines below work:

#pt._run_hwm93()                                                                
#pt._run_hwm07()                                                                
#pt._run_hwm14()
pt.run_hwm(version=1993)
pt.run_hwm(version=2007)
pt.run_hwm(version=2014)
pt.run_hwm()

**Comment by bharding512** [comment]
I wonder if this is resolved by the pull request (https://github.com/timduly4/pyglow/pull/109) that seems to have not been merged. Suggestions are welcome.

**Comment by sandyfreelance** [comment]
Great find, yes, that's exactly the fix for the two items.  Hadn't thought to check the pull requests, that one would close this issue.

**Comment by bharding512** [comment]
I failed to stay on top of this. @sandyfreelance if you've tested PR #109 let me know and I will accept it.

**Comment by sandyfreelance** [comment]
Yes, the code fix in #109 works and closes this #102 (and closes #109 as well)!

**Comment by bharding512** [comment]
Thanks, just squashed and merged into master. If this doesn't solve the problem please reopen.
